scripts/backend.ps1

```diff
@@ -174,6 +180,8 @@ class Backend {
 
 class AzureBackend : Backend {
     [String] $Name = "AzureBackend"
+    [String] $SecretsPath = "C:\Framework-Scripts\secrets.ps1"
+    [String] $CommonFunctionsPath = "C:\Framework-Scripts\common_functions.ps1"
```

Comment:
not again, please.
NEVER add code functionality as a parameter to something.

Reply:
git rebase :/ sorry, will remove it